src/lib/components/common/Button.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  export let href: string = null;
+  export let primary = false;
+  export let ghost = false;
+  export let disabled = false;
+
+  export let title: string = "";
+  export let type: "submit" | "reset" | "button" = "button";
+  export let label = "Submit";
+</script>
+
+{#if href}
+  <a {href} {title} on:click class:ghost class:disabled class:primary>
+    <slot>{label}</slot>
+  </a>
+{:else}
+  <button {title} on:click class:ghost class:primary {disabled} {type}>
+    <slot>{label}</slot>
+  </button>
+{/if}
+
+<style>
+  a,
+  button {
+    cursor: pointer;
+    display: inline-flex;
+    padding: 0.375rem 0.75rem;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
+
+    border-radius: 0.5rem;
+    border: 1px solid var(--gray-4, #5c717c);
+    background: var(--gray-3, #99a8b3);
+    box-shadow: 0px 2px 0px 0px #5c717c;
+
+    color: var(--gray-1, #f5f6f7);
+    text-align: center;
```

Comment:
Browser will default `button` elements to system font, so need to specify it:

```suggestion
    text-align: center;
    font-family: var(--font-sans, "Source Sans Pro");
```